### PR TITLE
Add ChannelSplitterNode

### PR DESCRIPTION
**Pull request by chrisdavidmills**
I've got one thing here that I'm not sure about — on https://developer.mozilla.org/en-US/docs/Web/API/ChannelSplitterNode there is a note. How do you think I should add this?

### CODE REVIEW

File: api/ChannelSplitterNode.json [hunk 1]
```diff
@@ -0,0 +1,107 @@
...
+  "api": {
+    "ChannelSplitterNode": {
+      "__compat": {
+        "mdn_url": "https://developer.mozilla.org/docs/Web/API/ChannelSplitterNode",
+        "support": {
+          "webview_android": {
+            "version_added": true
+          },
+          "chrome": {
+            "version_added": "14"
```
Comment (teoli2003) [inline api/ChannelSplitterNode.json]: Chromes note is missing (all chromes + opera): [1] Starting in Chrome 56, channelCountMode is set to "explicit" and channelCount is fixed to the number of outputs, as per the latest spec.

File: api/ChannelSplitterNode.json [hunk 2]
```diff
@@ -0,0 +1,107 @@
...
+          },
+          "edge": {
+            "version_added": true
+          },
+          "edge_mobile": {
+            "version_added": true
+          },
+          "firefox": {
+            "version_added": "25"
```
Comment (teoli2003) [inline api/ChannelSplitterNode.json]: Has similar changes to the one of Chrome already happened in Firefox? If so -> note.

File: api/ChannelSplitterNode.json (hunk 1 above)
Comment (chrisdavidmills) [reply api/ChannelSplitterNode.json]: Cool, added.

File: api/ChannelSplitterNode.json (hunk 2 above)
Comment (chrisdavidmills) [reply api/ChannelSplitterNode.json]: No, I don't think so.

**Pull request merged**